templates/console-server/020-ClusterRole-console-server.yaml

```diff
@@ -11,3 +11,6 @@ rules:
   - apiGroups: [ "" ]
     resources: [ "namespaces" ]
     verbs: [ "get", "list", "watch" ]
+  - apiGroups: [ "authorization.k8s.io" ]
+    resources: [ "subjectaccessreviews" ]
```

Comment:
what part does this play?  I don't see a change making a subjectaccessreviews.

Reply:
It was something I added earlier as I was changing it to use subjectaccessreviews when using impersonation. However, that seems to no longer be necessary, so I've removed that now. THanks!